cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java

```diff
@@ -66,6 +66,8 @@ public SpaceDevelopersOperations(DefaultCloudFoundryOperations cfOperations) {
      * @throws IllegalArgumentException when username is empty
      */
     public void assignSpaceDeveloper(String username) throws CreationException {
+        Log.debug("Assign a space developer:", username);
```

Comment:
No need for an imperative style. I'd suggest "Assigning space developer %s".

Reply:
Done